config/locales/frab.fr.yml

```diff
@@ -0,0 +1,158 @@
+fr:
+  actions: "Actions"
+  add_association: "Ajouter %{name}"
+  basic_information: "Informations de base"
+  call_for_participation: "Appel à participation"
+  classifiers: "Classificateurs"
+  create_new_conference: "Créer une nouvelle conférence"
+  create_your_first_conference: "Créez votre première conférence"
+  days: "Jours"
+  date_not_set: "Date non définie pour le moment"
+  edit: "Éditer"
+  edit_conference: "Éditer la conférence"
+  edit_conference_classifiers: "Éditer les classificateurs de la conférence"
+  edit_conference_days: "Éditer les jours de la conférence"
+  edit_conference_rooms: "Éditer les salles de la conférence"
+  edit_conference_schedule: "Modifier la programmation de la conférence"
+  edit_conference_tracks: "Modifier les canaux de la conférence"
+  edit_ticket_server: "Éditer le serveur de tickets"
+  error_signing_in: "Email ou mot de passe invalide."
+  event_languages: "Langues de l'évènement"
+  features: "Caractéristiques additionnelles" # "Additional features"
+  new_conference: "Nouvelle conférence"
+  notifications: "Notifications"
+  participate: "Participer"
+  remove_association: "Enlever %{name}"
+  rooms: "Pièces"
+  schedule: "Programme" # "Schedule"
+  show: "Afficher"
+  mail_templates: "Templates d'emails"
+  settings: "Paramètres"
+  sign_in_successful: "Identifié avec succès."
+  submission_configuration: "Configuration des propositions"
+  ticket_server: "Serveur de tickets"
+  timeslot_configuration: "Configuration de l'intervalle de temps"
+  tracks: "Canaux" # "Tracks"
+  your_submission: "Votre proposition"
+
+  navigation:
+    event: "Détail de l'évènement"
+    people: "Personnes" # "People"
+    ratings: "Notes"
+    feedbacks: "Retours utilisateurs"
+    conferences: "Conférences"
+    users: "Utilisateurs"
+
+  ability:
+    denied: "Cette action est interdite."
+
+  cfp:
+    login: "Connexion"
+
+  events: "Évènements"
+
+  conflict:
+    events_overlap: "se chevauche avec"
+    has_no_availability: "n'a pas encore entré ses disponibilités."
+    person_unavailable: "n'est pas disponible dans la plage prévue."
+    person_already_speaking: "parle déjà à"
+
+  options:
+    business: "business"
+    concert: "concert"
+    dect: "DECT" # pas changé
```

Comment:
Remove comment

Reply:
ok, but what is a DECT ? :)

Reply:
[Some kind of mobile phone](https://fr.wikipedia.org/wiki/DECT_(t%C3%A9l%C3%A9phone_sans_fil)) apparently.  Not sure in what way it's different than a "mobile".